### ISSUE TITLE
Interdyne Miners are unfair competition and unclear in scope

### ISSUE THREAD

**Issue by YakumoChen** [Skyrat-SS13/Skyrat-tg#4123]
Lavaland syndie miners are a negative impact to Lavaland and create bad conflict with other Lavaland roles.

-Their spawn text is unclear because they are lazily copy-pasted from the Scientist role. As miners they're naturally required to _leave_ the base and "laying low" is intentionally difficult by their very nature. They are direct competition with Station Miners for resources and almost never make any actual effort to blend in because their provided gear is flat out better than miner gear (they get syndie hardsuits)

-Miners are more or less forbidden to get into fights with them, even though they are, again, direct competition. They may more or less freely deny """"miner loot""" like megafauna kills and tendril chests to station players. There is no point in engaging in conflict anyway with syndie miners, because:

-Conflict with miners is one-sided and unfair. Lavaland syndies get sniper rifles, which are obscenely powerful, and wear hardsuits which have superb armor values, on top of whatever lavaland loot they manage to take first.

-They don't provide any benefit to the ongoing round. Given they no longer can share a channel with Traitors, they may as well be playing their own game even though they share the same space.

**Comment by Draganfrukts** [comment]
>  -Miners are more or less forbidden to get into fights with them, even though they are, again, direct competition. They may more or less freely deny """"miner loot""" like megafauna kills and tendril chests to station players.

I don't see the issue with this as I never saw Interdyne miners aggravating station miners and vice versa. One of two gets scared at the sight of drake armor/HECK suit/explorer suit as the other gets scared of seeing funny red man. PKA explosions still deal their ridiculous 40 damage at a delay much lower than the sniper rifle's _60 deciseconds_

>  -Conflict with miners is one-sided and unfair. Lavaland syndies get sniper rifles, which are obscenely powerful, and wear hardsuits which have superb armor values, on top of whatever lavaland loot they manage to take first.

PKAs do not give half a shit about armor. Sniper rifles have a 6 second delay between shots. The blood-red hardsuit's melee armor is tame compared to what miners can do.

>  -They don't provide any benefit to the ongoing round. Given they no longer can share a channel with Traitors, they may as well be playing their own game even though they share the same space.

Yet we still have Cybersun and DS-1. And golems. And hermits. :okbuddy:



TL;DR: Not broken, not worth fixing if you could.

**Comment by YakumoChen** [comment]
> I don't see the issue with this as I never saw Interdyne miners aggravating station miners and vice versa. One of two gets scared at the sight of drake armor/HECK suit/explorer suit as the other gets scared of seeing funny red man. PKA explosions still deal their ridiculous 40 damage at a delay much lower than the sniper rifle's _60 deciseconds_
> PKAs do not give half a shit about armor. Sniper rifles have a 6 second delay between shots. The blood-red hardsuit's melee armor is tame compared to what miners can do.

1) PKAs have extremely limited range
2) Sniper rifle shots stun on hit and typically delimb
3) Players are discouraged from interacting because of The Rules yet the syndieminers don't serve any purpose except to let players who didn't roll miner a way to speedrun lavaland anyway.  Bioweapon sci (the tutorial roles the ghost role serves as a basis for) doesn't actually need mining to operate and only is given a lathe and tech "just because"


> Yet we still have Cybersun and DS-1. And golems. And hermits. :okbuddy:
> TL;DR: Not broken, not worth fixing if you could.

Cybersun actually made more sense to separate since they served a pretty strong ally to traitors and were much easier to interact with. They also inhabit deep space which is much more disconnected from the station (lavaland ALWAYS has miners and is vital to station operations, deep space has to be intentionally visited and contains only oddities)

DS1 is equivalent to nukies and operates on the _fucking Centcomm level_ so it's guaranteed to never interact with the game round except on Assualt Ops

Other lavaland roles actually have healthy interactions with the station and aren't just Overpowered Miner Speedrunners for Bored Ghosts

Lavaland syndies had actually interesting dynamics with traitors that were removed for No Reason. The lava moat meant traitors couldn't just stroll up to the base and visit and provided interesting support options with the comms agent and bioweapons researchers, the added mining is just a clumsy addition that does more harm than good.

tldr; Syndie miners not needed, revert the role and seal up the base and/or give traitor comms back
